pandas/core/ops.py

```diff
@@ -1115,7 +1115,7 @@ def dispatch_to_series(left, right, func, str_rep=None, axis=None):
     import pandas.core.computation.expressions as expressions
 
     right = lib.item_from_zerodim(right)
-    if lib.is_scalar(right):
+    if lib.is_scalar(right) or np.ndim(right) == 0:
```

Comment:
why is this needed? this *should* already be handled by is_scalar

Reply:
because there are lots of weird things a user _could_ pass that are zero-dim but not caught by is_scalar.  The test case that hits this just passes `object()`